db/view/view_updating_consumer.hh

```diff
@@ -83,7 +83,7 @@ public:
 
     void consume_new_partition(const dht::decorated_key& dk) {
         _mut_builder.emplace(_schema);
-        _mut_builder->consume_new_partition(dk);
+        _buffer_size += _mut_builder->consume_new_partition(dk).memory_usage(*_schema);
     }
```

Comment:
This is still inaccurate: this memory_usage() will change as rows are consumed, since mutation_partition::_rows consumes more memory (which isn't accounted for by consume(clustering_row).

I don't see an efficient way to fix it, and we can live with the inaccuracy. But please add a comment about it.

Reply:
I already noted this in the commit log, will add a comment also.

Reply:
done in v2.